README.md

```diff
@@ -29,10 +33,14 @@ to the [Contributor License Agreement](https://www.w3.org/community/about/agreem
 
 ## Contributions
 
-You are not required to be a member of the group in order to
+You are not required to be a member of the Community Group or Working Group in order to
 [file issues](https://github.com/gpuweb/gpuweb/issues), errors, fixes or make suggestions.
-Just a github account. We simply require that any significant contribution of technology
-come from members, so that we can ensure no IP complications down the line.
+Anyone with a GitHub account can do so.
+
+In order to make a "significant" contribution to WebGPU potentially containing
+Intellectual Property, you must be either a member of the Working Group, or a
+member of the Community Group who has signed the
+[Final Specification Agreement](https://www.w3.org/community/about/process/fsa-deed/).
```

Comment:
Actually, the group can also accept "significant" contributions from a person who is neither WG nor CG participants. In that case, we will ask the person to make a non-participant commitment (essentially the same thing as signing the Final Specification Agreement). This is described in the [Licence Grants from Non-Participants](https://www.w3.org/2021/Process-20211102/#contributor-license) section in the process.

```suggestion
In order to make a "significant" contribution to WebGPU potentially containing
Intellectual Property, you must be either a member of the Working Group, a
member of the Community Group who has signed the
[Final Specification Agreement](https://www.w3.org/community/about/process/fsa-deed/),
or you must provide a royalty-free patent commitment on the terms specified in the
[W3C Royalty-Free (RF) Licensing Requirements](https://www.w3.org/Consortium/Patent-Policy-20200915/#sec-W3C-RF-license)
section of the W3C Patent Policy.
```

Or you may want to turn that around with something like:

"In order to assure that WebGPU specifications can be implemented on a Royalty-Free (RF) basis, all significant contributions need to be made with RF commitments. Members of the Working Group, and members of the Community Group who have signed the [Final Specification Agreement](https://www.w3.org/community/about/process/fsa-deed/), already committed to the terms of the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/). Non-members will be requested to provide an RF commitment under similar terms."

Reply:
I really like your "turned around" version, I will use it!